src/pages/[...slug].tsx

```diff
@@ -0,0 +1,194 @@
+import { Box, Divider as ChakraDivider, Flex, Heading, Text, chakra } from "@chakra-ui/react"
+import { ParsedUrlQuery } from "querystring"
+import { MDXRemote } from "next-mdx-remote"
+import { serialize } from "next-mdx-remote/serialize"
+import remarkGfm from "remark-gfm"
+
+import InfoBanner from "@/components/InfoBanner"
+import Link from "@/components/Link"
+import MarkdownTable from "@/components/MarkdownTable"
+
+import { getContent, getContentBySlug } from "@/lib/utils/md"
+
+import { GetStaticPaths, GetStaticProps, NextPage } from "next/types"
+import { ChildOnlyProp } from "@/lib/types"
```

Comment:
Not sure if this has any performance impact on Next (it has on Gatsby) but we did specified that these were imported `type`s.

```suggestion
import type { GetStaticPaths, GetStaticProps, NextPage } from "next/types"
import type { ChildOnlyProp } from "@/lib/types"
```